src/pages/productListing/ProductListing.jsx

```diff
@@ -0,0 +1,112 @@
+import "./ProductListing.css";
+import { dumbellPI } from "../../images/index";
+import { Card } from "../../components/index";
+
+export default function ProductListing() {
+  return (
+      <>
+    <main className="product-listing-page">
+      <aside className="filters-container p-xs">
```

Comment:
You should extract out this Filter container separately as a component, code would look much better.

Reply:
Yes, even i realised it, but this PR already has a lot of changes, hence thought of refactoring it in the next one.

Reply:
Okay no problem, I just wanted to inform you about the same.